.generator/src/generator/templates/http/isomorphic-fetch.j2

```diff
@@ -7,6 +7,10 @@ import { isBrowser, isNode } from "../util";
 export class IsomorphicFetchHttpLibrary implements HttpLibrary {
   public debug = false;
   public zstdCompressorCallback: ZstdCompressorCallback | undefined;
+  public enableRetry!: boolean;
+  public maxRetries!: number ;
+  public backoffBase!: number ;
+  public backoffMultiplier!: number;
```

Comment:
What does the bang do here? Makes sure its not undefined?

Reply:
Yep the exclamation marks are there for syntax reasons because those params are not initialised in this class but the compiler doesn't know and consider this an error without `!`